systemtests/bin/download_sym_files.py

```diff
@@ -14,6 +14,20 @@
 import markus
 from markus.backends import BackendBase
 import requests
+from unittest.mock import ANY
+
+# These are exclusively security headers added by nginx
+TECKEN_RESPONSE_HEADERS = {
+    "X-Content-Type-Options": "nosniff",
+    "X-Frame-Options": "DENY",
+    "Content-Security-Policy": "font-src 'self'; object-src 'none'; script-src 'self'; img-src 'self'; default-src 'self'; style-src 'self'; connect-src 'self'; frame-ancestors 'none'",
```

Comment:
While the new HEAD request checks pass, there is one response header check that is failing for AWS stage and prod: `Content-Security-Policy` expected versus actual do not match for the Tecken redirect response. The difference is that the actual header response has: `; report-uri /__cspreport__`.

MDN [indicates](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/report-uri) that this `report-uri` directive is deprecated and no longer recommended in favor of `report-to`, but `report-to` isn't yet supported in all browsers. Should we include both the `report-uri` and `report-to` directives? Do we still want these directives?

Also who looks at these reports?